### PR TITLE
Planning Service

### PR DESCRIPTION
**Pull request by thiago-rocha**

### CODE REVIEW

File: PontoFacil/PontoFacil/Services/IPlanningService.cs [hunk 1]
```diff
@@ -1,6 +1,11 @@
-﻿namespace PontoFacil.Services
+﻿using PontoFacil.Models;
+
+namespace PontoFacil.Services
 {
     public interface IPlanningService
     {
+        Planning GetPlanning();
+
```
Comment (soutolucas) [inline PontoFacil/PontoFacil/Services/IPlanningService.cs:8]: This line is unnecessary.

File: PontoFacil/PontoFacil/Models/Planning.cs [hunk 2]
```diff
@@ -1,6 +1,38 @@
...
+
+        private int _numberOfDaysValue;
+        public int NumberOfDaysValue
+        {
+            get { return _numberOfDaysValue; }
+            set { SetProperty(ref _numberOfDaysValue, value); }
+        }
+
+        private System.Nullable<DateTimeOffset> _startDayValue;
+        public System.Nullable<DateTimeOffset> StartDayValue
```
Comment (soutolucas) [inline PontoFacil/PontoFacil/Models/Planning.cs:31]: The namespace System already imported in line 2

File: PontoFacil/PontoFacil/Services/PersistencyService.cs [hunk 3]
```diff
@@ -99,19 +98,20 @@ public Profile SaveProfile(Profile profile)
...
         public ClockIn getClockInById(DateTime datetime)
         {
-            
             return this._repository.ClockInList.Find(ci => ci.Id.Equals(datetime));
         }
 
         public Planning getPlanning()
         {
             Planning p = this._repository.MyPlanning;
+
```
Comment (soutolucas) [inline PontoFacil/PontoFacil/Services/PersistencyService.cs:107]: This line is unnecessary.

File: PontoFacil/PontoFacil/Services/PersistencyService.cs [hunk 4]
```diff
@@ -99,19 +98,20 @@ public Profile SaveProfile(Profile profile)
...
             return p != null ? p : new Planning();
         }
 
         public Profile getProfile()
         {
             Profile p = this._repository.MyProfile;
+
```
Comment (soutolucas) [inline PontoFacil/PontoFacil/Services/PersistencyService.cs:114]: This line is unnecessary.

File: PontoFacil/PontoFacil/Services/PlanningService.cs [hunk 5]
```diff
@@ -1,12 +1,30 @@
...
     public class PlanningService : IPlanningService
     {
+        #region Properties
+        private IPersistencyService _persistencyService;
+        #endregion
+
+        #region Constructor
+        public PlanningService(IPersistencyService persistencyService)
+        {
+            this._persistencyService = persistencyService;
```
Comment (soutolucas) [inline PontoFacil/PontoFacil/Services/PlanningService.cs]: The keyword "this" is unnecessary.

File: PontoFacil/PontoFacil/Services/PlanningService.cs [hunk 6]
```diff
@@ -1,12 +1,30 @@
...
+        }
+        #endregion
+
+        #region Methods
+        public Planning GetPlanning()
+        {
+            return this._persistencyService.getPlanning();
```
Comment (soutolucas) [inline PontoFacil/PontoFacil/Services/PlanningService.cs]: The keyword "this" is unnecessary.

**Review by soutolucas — CHANGES_REQUESTED** [review]
Please, look the single comments.

**Review by gmojunior — APPROVED** [review]

**Comment by thiago-rocha** [comment]
Ready to review @soutolucas

**Review by soutolucas — APPROVED** [review]

**Pull request merged**